packages/mds-agency/utils.ts

```diff
@@ -263,7 +293,7 @@ export function badTelemetry(telemetry: Telemetry | null | undefined): ErrorObje
 }
 
 // TODO Joi
-export async function badEvent(event: VehicleEvent) {
+export async function badEvent(device: Pick<Device, 'modality'>, event: VehicleEvent) {
```

Comment:
`device` is expected to be an object with just a `modality` property here?

Reply:
Yeah, we basically only need the `modality` to be passed in. Everywhere this is used currently a fully qualified Device object is passed in, but I could imagine us wanting to try and reduce object size, so it felt nice to specify we _only_ needed the modality property to be defined.